hawk/src/main/java/com/orhanobut/hawk/Utils.java

```diff
@@ -22,4 +22,11 @@ public static void checkRx() {
           "make sure that you have it in your dependencies");
     }
   }
+
+  public static void validate() {
```

Comment:
Ohh, sorry by mistake, I deleted the message. 

About the name, we can refer the build process. maybe validateBuild?
```java
validateBuild()
```